Modules/CTP/include/CTP/TH1ctpReductor.h

```diff
@@ -0,0 +1,80 @@
+// Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+// See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+// All rights not expressly granted are reserved.
+//
+// This software is distributed under the terms of the GNU General Public
+// License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+///
+/// \file   TH1ctpReductor.h
+/// \author Lucia Anna Tarasovicova
+///
+#ifndef QUALITYCONTROL_TH1CTPREDUCTOR_H
+#define QUALITYCONTROL_TH1CTPREDUCTOR_H
+
+#include "QualityControl/Reductor.h"
+
+namespace o2::quality_control::postprocessing
+{
+
+/// \brief A Reductor which obtains the characteristics of TH1 including the bin contents.
+/// Based on the example by Piotr Jan Konopka
+/// It produces a branch in the format: "mean/D:stddev:entries:inputs[%i]:classes[%i]"
+class TH1ctpReductor : public quality_control::postprocessing::Reductor
```

Comment:
```suggestion
class TH1ctpReductor : public quality_control::postprocessing::ReductorTObject
```
There was a change in the framework while you were working on your PR. Essentially, you have to inherit now `ReductorTObject`. You may have a look how I adapted the tasks which were already in the codebase: https://github.com/AliceO2Group/QualityControl/pull/2110/files